app/helpers/application_helper.rb

```diff
@@ -6,4 +6,14 @@ def identifier_types
   def percentage(ratio)
     "#{(ratio * 100).round}%"
   end
+
+  def github_search_enabled?
+    ENV['GITHUB_ORGANIZATION'].present?
```

Comment:
can you add a note to the README about this feature? 

it would be useful to document this as a configurable env variable that enables this functionality and what it's useful for.